src/main/java/net/dv8tion/jda/core/entities/impl/GuildImpl.java

```diff
@@ -313,6 +313,7 @@ public Role getPublicRole()
     }
 
     @Override
+    @Deprecated
     public TextChannel getPublicChannel()
```

Comment:
Since this is going to be in 3.4 we will be removing this method anyway FYI

Reply:
just added it because gradle build complained about implementing a deprecated method